src/object_store/cache.rs

```diff
@@ -419,8 +419,8 @@ mod tests {
     #[case::fetch_nothing_at_the_chunk_boundary(16..16)]
     #[case::fetch_one_byte_at_the_chunk_boundary(16..17)]
     #[case::part_of_chunk_0_part_of_chunk_1(10..20)]
-    #[case::reaches_the_end_of_the_body(10..500)]
-    #[case::goes_beyond_the_end_of_the_body(400..505)]
+    #[case::reaches_the_end_of_the_body(10..484)]
```

Comment:
Wait, what happened here? Did something change in the `object_store`?

Reply:
Good question, I'm not quite sure. My initial impression is that something changed in the (`ArrowWriter` of the) `parquet` crate, but didn't dig into it further.

Reply:
Ah, makes sense. I forgot this was using a Parquet file for tests.